crates/release_plz/src/main.rs

```diff
@@ -82,7 +87,39 @@ async fn run(args: CliArgs) -> anyhow::Result<()> {
     Ok(())
 }
 
-fn get_release_pr_req(
+fn get_release_pr_reqs(
+    config: &Config,
+    update_request: UpdateRequest,
+) -> anyhow::Result<Vec<ReleasePrRequest>> {
+    if config.workspace.one_pr_per_package.unwrap_or(false)
+        && update_request.single_package().is_none()
+    {
+        let packages = update_request.cargo_metadata().workspace_packages();
+
+        packages
+            .into_iter()
+            .map(|package| {
+                let update_request = update_request
+                    .clone()
+                    .with_single_package(package.name.clone());
+
+                let release_pr_request = get_single_release_pr_req(config, update_request)?;
+
+                let branch_prefix = release_pr_request.branch_prefix().to_owned();
+
+                let release_pr_request = release_pr_request
+                    .with_branch_prefix(Some(format!("{branch_prefix}{}-", package.name)));
```

Comment:
```suggestion
                    .with_branch_prefix(Some(format!("{branch_prefix}-{}", package.name)));
```
this is a typo, right?

Reply:
Not really. From the documentation for the `branch_prefix` configuration option, I infer that the prefix contains whatever separator character is desired at the end, which is why there is no dash between the prefix and the package name. The '-' is a suffix for the package name to separate it from the timestamp.

I am open to suggestions on how that can be made more clear